tests/planner_test/count.planner.sql

```diff
@@ -2,19 +2,22 @@
 explain select count(*) from t
 
 /*
-
-Projection { cost: 302.3, exprs: [ rowcount ] }              
-└── Aggregate { aggs: [ rowcount ], cost: 301, group_by: [] }
-    └── Scan { cost: 0, list: [], table: t }
+Projection { exprs: [ rowcount ], cost: 302.3 }
+└── Aggregate                                  
+    ├── aggs: [ rowcount ]                     
+    ├── group_by: []                           
+    ├── cost: 301                              
+    └── Scan { table: t, list: [], cost: 0 }
```

Comment:
Why is this changed? It seems you didn't change the line width and only reordered the fields. 🤔

Reply:
Good question!